Disable flaky tests

soulitzer opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #55871 Refactor fast gradcheck
* #55699 Enable fixed fast_mode for complex
* #55692 Refactor C->C to C->R twice
* #55656 Clean up raise exception logic
* #55237 Enable fast gradcheck for real inputs and outputs
* **#56279 Disable flaky tests**

Differential Revision: [D27916606](https://our.internmc.facebook.com/intern/diff/D27916606)

soulitzer committed:
Disable flaky tests

[ghstack-poisoned]

albanD commented on test/test_nn.py:
```diff
@@ -7867,7 +7867,7 @@ def _gelu_ref(X):
                 ref = _gelu_ref(X.to(numpy_dtype).cpu().detach().numpy())
                 self.assertEqual(res, ref, rtol=rtol, atol=atol)
-                if dtype != torch.bfloat16:
+                if dtype == torch.float64:
```
What is this change?

soulitzer replied:
_test_gelu is only being called with bfloat16, float32, and float64. This change just makes it so that we won't test float32 for gradcheck anymore

albanD commented on test/test_linalg.py:
```diff
@@ -2916,7 +2916,8 @@ def func(root, b, upper):
             # TODO(#50743): the following fails with batched grad testing
-            gradgradcheck(func, [root, b, upper], atol=1e-3, check_batched_grad=False)
+            # TODO(#56235): disabling temporarily
+            # gradgradcheck(func, [root, b, upper], atol=1e-3, check_batched_grad=False)
```
Does raising atol to 1e-2 work? That might be better than completely removing it?

soulitzer replied:
Not quite, for certain seeds the difference can be quite large. I have more details in the linked issue, but basically for 90% of the seeds I sampled, the difference is small i.e., it passes with atol=1e-12, rtol=1e-3. However, there are certain seeds where the randomized inputs produce differences very large as well. So no matter what atol, we set, there would always be a non-negligible chance that it fails.

soulitzer committed:
Update on "Disable flaky tests"


Differential Revision: [D27916606](https://our.internmc.facebook.com/intern/diff/D27916606)

[ghstack-poisoned]

soulitzer committed:
Update on "Disable flaky tests"


Differential Revision: [D27916606](https://our.internmc.facebook.com/intern/diff/D27916606)

[ghstack-poisoned]
